.gitlab-ci.yml

```diff
@@ -0,0 +1,65 @@
+image: python:3.6
+
+stages:
+  - infra
+  - apps
+  - test
+  - deploy
+
+before_script:
+  - echo "Running Before Script"
+  - source environment
+  - apt-get -y update
+  - apt-get -y install jq moreutils curl unzip zip
+  - mkdir ./external_binaries
+  - curl https://releases.hashicorp.com/terraform/0.11.7/terraform_0.11.7_linux_amd64.zip -o /tmp/terraform.zip
+  - unzip /tmp/terraform.zip -d ./external_binaries/
+  - rm /tmp/terraform.zip
+  - export PATH=$PATH:`pwd`/external_binaries/
+  - pip install awscli --upgrade
+  - pip install boto3 --upgrade
+  - pip install virtualenv
+  - source environment
+  - make secrets
+
+infra:
+  stage: infra
+  only:
+    - gitlab
+    - master
```

Comment:
I think you'd want to run plan in PRs too right?

Reply:
👍